### PR TITLE
Added pagination for homepage and category pages (limited to 15 per page)

### PR DESCRIPTION
**Pull request by BrandonBalala**
Resolves [User Story 4] #177

### CODE REVIEW

File: app/Http/Controllers/HomeController.php [hunk 1]
```diff
@@ -52,8 +52,7 @@ public function index(Request $request)
         }
 
         $products = Product::orderBy($sortBy_column, $orderBy)
-        ->take(25)
-        ->get();
+                            ->paginate(15);
```
Comment (kevcamel) [inline app/Http/Controllers/HomeController.php:55]: github always seems to mess up indentation

File: resources/views/categoryPage.blade.php [hunk 2]
```diff
@@ -99,43 +99,17 @@
...
-			<!-- Next -->
-			<li class="page-item">
-				<a class="page-link" href="#">&rsaquo;</a>
-			</li>
-			<!-- Next -->
-		</ul>
-	</nav>
-	<!-- END PAGINATION -->
+	<!-- Pagination -->
+	{{ $products->links('vendor.pagination.bootstrap-4') }}
```
Comment (kevcamel) [inline resources/views/categoryPage.blade.php:108]: ouf, view jsut for the pagination, very nice 👌

**Review by kevcamel — COMMENTED** [review]
Code is clean. Indentation is good, except for where Github messes it up. Commenting is good too, keep it up 👍

**Comment by kcamcam** [comment]
@BrandonBalala what do we do for the acceptance test?

**Comment by neqqash** [comment]
well done, 15 per page is a good number, not too much and not too little. The code is really simple to follow as well.

File: resources/views/categoryPage.blade.php (hunk 2 above)
Comment (BrandonBalala) [reply resources/views/categoryPage.blade.php:108]: super easy to implement!

**Pull request merged**